visualstudio/2017/netstandard/NetstandardModuleSample/NetstandardModuleSample/module_dev_sample.json

```diff
@@ -0,0 +1,55 @@
+﻿{
+    "loaders": [
+        {
+            "type": "dotnetcore",
+            "name": "dotnetcore",
+            "configuration": {
+                "binding.path": "dotnetcore.dll",
+                "binding.coreclrpath": "C:\\Program Files\\dotnet\\shared\\Microsoft.NETCore.App\\1.1.1\\coreclr.dll",
```

Comment:
Shall we put hard coded path here, or we let runtime to load it from default location ?